src/android/InAppBrowser.java

```diff
@@ -922,6 +923,22 @@ public void run(){
         public InAppBrowserClient(CordovaWebView webView, EditText mEditText) {
             this.webView = webView;
             this.edittext = mEditText;
+            this.allowedSchemes = preferences.getString("AllowedSchemes", "").split(",");
+        }
+
+        /**
+         * Handles custom url schemes by sending customscheme events the local browser
+         */
+        public boolean handleCustomUrlScheme(String url) {
```

Comment:
If you can: 

- make private
- rename to `isWhitelistedCustomScheme`
- move the `browserEventSender.customScheme(url);` into the calling code

This gives a clearer intent and makes the function do one thing: test for valid whitelisted scheme.